benchkit/__main__.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+from sanic import Sanic
+from sanic.worker.loader import AppLoader
+
+from .app import create_app
+from .env import env
+
+
+if __name__ == '__main__':
+    loader = AppLoader(factory=create_app)
+    app = loader.load()
+    # app.prepare(port=env.backend_port, debug=True, workers=1, dev=True)
```

Comment:
Is it a leftover?

Reply:
No, it's for local development. The server will reload changed python files with this config. Probably worth a comment :upside_down_face: thanks.